strategies/affected/src/main/java/org/arquillian/smart/testing/strategies/affected/AffectedRunnerProperties.java

```diff
@@ -20,9 +25,23 @@
     static final String EXCLUSIONS = "exclusions";
 
     private final Properties properties = new Properties();
+    private AffectedConfiguration affectedConfiguration;
 
-    AffectedRunnerProperties(){
-        readFile(System.getProperty(SMART_TESTING_AFFECTED_CONFIG));
+    AffectedRunnerProperties(File rootDirectory) {
+        final Configuration configuration = Configuration.loadPrecalculated(rootDirectory);
```

Comment:
you shouldn't load the precalculated configuration here - you should use that one that was set by ST API in the surefire provider.

Reply:
any good reason for it?

Reply:
The reason is this issue I fixed some time ago: https://github.com/arquillian/smart-testing/issues/231

Reply:
> Because we are doing the same in it 

no, we are not - we provide the instance of the configuration as a parameter: https://github.com/arquillian/smart-testing/blob/1498d4051942f3b8725712bcd5fbe547cbd2e82b/surefire-provider/src/main/java/org/arquillian/smart/testing/surefire/provider/SmartTestingSurefireProvider.java#L81
and now imagine that I need to call the ST API from a different directory than the project root directory is.